### ISSUE TITLE
dist/preparetxpidist.sh: Provide standard wpa_supplicant.conf

### ISSUE THREAD

**Issue by heuer** [ftCommunity/tx-pi#47]
Currently, the script deletes the file.
Maybe we should provide a useful standard configuration

```
ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
update_config=1
country=DE
```

It contains no sensitive data, just the (default) country code.

**Comment by PeterDHabermehl** [comment]
Agree.-- Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 01.05.20, 22:19 schrieb Lars <notifications@github.com>:

   
  Currently, the script deletes the file. Maybe we should provide a useful standard configuration 
  ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
update_config=1
country=DE
 
  It contains no sensitive data, just the (default) country code.

**Comment by heuer** [comment]
I was wrong, the script provided a default configuration but the country code was missing.

Fixed by e7e40ed
